recipe/meta.yaml

```diff
@@ -10,43 +9,58 @@ package:
 source:
   fn: RandomFields_{{ version }}.tar.gz
   url:
-    - https://cran.r-project.org/src/contrib/RandomFields_{{ version }}.tar.gz
-    - https://cran.r-project.org/src/contrib/Archive/RandomFields/RandomFields_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/RandomFields_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/RandomFields/RandomFields_{{ version }}.tar.gz
   sha256: 2d6a07c3a716ce20f9c685deb59e8fcc64fd52c8a50b0f04baf451b6b928e848
 
 build:
-  number: 0
-  script: R CMD INSTALL --build .
```

Comment:
@cbrueffer you need to restore this line or add the `build.sh` and `bld.bat` scripts.

Reply:
Oops, good spotting! Let me fix this...